bot/exts/utilities/bookmark.py

```diff
@@ -42,51 +81,59 @@ def build_bookmark_dm(target_message: discord.Message, title: str) -> discord.Em
         return embed
 
     @staticmethod
-    def build_error_embed(user: discord.Member) -> discord.Embed:
-        """Builds an error embed for when a bookmark requester has DMs disabled."""
+    def build_error_embed(message: str) -> discord.Embed:
+        """Builds an error embed for a given message."""
```

Comment:
'message' here feels like a discord message, but its just a string.

This may be worth raising another issue given how many different methods are like this in the bot. ERROR_REPLIES and Colours.soft_red make a lot of error embeds, that it might be worth putting this in a utils file now and standardizing them as they are come across.